functionaltests/src/test/java/util/WebDriverSofortueberweisung.java

```diff
@@ -41,8 +42,10 @@ public WebDriverSofortueberweisung(final String pin, final String tan) {
         timeouts.pageLoadTimeout(DEFAULT_TIMEOUT, TimeUnit.SECONDS);
         timeouts.setScriptTimeout(DEFAULT_TIMEOUT, TimeUnit.SECONDS);
 
+        manage().window().fullscreen();
```

Comment:
In some cases it is possible that button control is hidden in a virtual browser window, and selenium will complain the button can't be clicked! 

This is not a case right now, but it protects us from future fails if they change design and the button is located on the bottom or at the right.